source/fab/__main__.py

```diff
@@ -29,7 +29,9 @@ def parse_cli() -> argparse.Namespace:
     parser.add_argument('-v', '--verbose', action='store_true',
                         help='Produce a running commentary on progress')
     parser.add_argument('-w', '--workspace', metavar='FILENAME', type=Path,
-                        help='Directory for working files.')
+                        help='Directory for working files')
+    parser.add_argument('--fpp-flags', action='store', type=str, default='',
+                        help='Provide flags for Fortran PreProcessor ')
```

Comment:
Using an argument in lue of an actual configuration system is fine but can you add a "TODO" to note that it is a temporary fix.

Reply:
Good point